tests/integration/targets/test_azure_manage_network_interface/tasks/main.yml

```diff
@@ -85,3 +85,15 @@
         name: "{{ network_interface_virtual_network }}"
         resource_group: "{{ resource_group }}"
         state: absent
+
+    - name: Delete a Network Interface
```

Comment:
Why did you add this? wasn't it already deleted in "Delete Network interface" task?

Reply:
It was added to always section, to clean up in case of failure